src/index.html

```diff
@@ -11,6 +11,27 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="container">
+      <div class="search-bar search-bar__big"
+        data-qa="big"
+        >
```

Comment:
```suggestion
      <div class="search-bar search-bar__big" data-qa="big">
```
If you have two attributes write them in one line with tag and that should be form tag not a div

Reply:
this is not fixed